pages.de/common/find.md

```diff
@@ -0,0 +1,36 @@
+# find
+
+> Dateien oder Verzeichnisse in einem Verzeichnisbaum rekursiv suchen.
+> Weitere Informationen: <https://manned.org/find>.
+
+- Dateien nach Erweiterung suchen:
+
+`find {{root_path}} -name '{{*.ext}}'`
+
+- Sucht nach Dateien, die mehreren Pfad-/Namensmustern entsprechen:
+
+`find {{root_path}} -path '{{*/path/*/*.ext}}' -or -name '{{*pattern*}}'`
+
+- Sucht nach Verzeichnissen, die einem bestimmten Namen entsprechen, ohne Berücksichtigung der Groß- und Kleinschreibung:
```

Comment:
```suggestion
- Sucht nach Verzeichnissen, die ohne Berücksichtigung der Groß- und Kleinschreibung einem bestimmten Namen entsprechen:
```

I think that this formulation is clearer because it needs less relative clauses. Also: why "Namen" here and "Namensmuster" above? Same issue in the english version, so this is fine here.